scalac-scoverage-plugin/src/main/scala/scoverage/Serializer.scala

```diff
@@ -33,9 +33,9 @@ object Serializer {
           <classType>
             {stmt.location.classType.toString}
           </classType>
-          <topLevelClass>
-            {stmt.location.topLevelClass}
-          </topLevelClass>
+          <fullClassName>
+            {stmt.location.fullClassName}
+          </fullClassName>
```

Comment:
I believe that the XML format used here is internal to `scoverage`, so can be changed without BC concerns